packages/app/prisma-utils/src/prismaError.ts

```diff
@@ -17,6 +17,9 @@ export const isPrismaClientKnownRequestError = (
 	typeof error.code === 'string' &&
 	error.code.startsWith('P')
 
+export const isPrismaTransactionClosedError = (error: PrismaClientKnownRequestError): boolean =>
+	error.code === PRISMA_TRANSACTION_ERROR && error.message.includes('Transaction already closed')
```

Comment:
There is no other way to recognize a timeout issue, in all my tests we always receive the same message (`Transaction already closed`).

We can't rely only on `PRISMA_TRANSACTION_ERROR` as, in the documentation they mention that it can be used for other purposes